tutorials/xr/setting_up_xr.rst

```diff
@@ -9,7 +9,12 @@ Introduction to the XR system in Godot
 Godot provides a modular XR system that abstracts many of the different XR platform specifics away from the user.
 At the core sits the :ref:`XRServer <class_xrserver>` which acts as a central interface to the XR system that allows users to discover interfaces and interact with the components of the XR system.
 
-Each supported XR platform is implemented as an :ref:`XRInterface <class_xrinterface>`. Supported interfaces register themselves with the :ref:`XRServer <class_xrserver>` and can be queried with the ``find_interface`` method on the :ref:`XRServer <class_xrserver>`. When the desired interface is found it can be initialized by calling ``initialize`` on the interface.
+Each supported XR platform is implemented as an :ref:`XRInterface <class_xrinterface>`.
+A list of supported platforms can be found on the list of features page :ref:`here <doc_xr_support>`.
+Supported interfaces register themselves with the :ref:`XRServer <class_xrserver>`
+and can be queried with the ``find_interface`` method on the :ref:`XRServer <class_xrserver>`.
+When the desired interface is found it can be initialized by calling ``initialize``
```

Comment:
Owh dang, this is a little outdated now. It's still correct and how it was designed to work, but for OpenXR we've had to early initialise in the core. While we talk about that elsewhere, wonder if we should add a note about it here.

Totally offtopic for VisionOs though so something to do separately.